scripts/overview.js

```diff
@@ -1,116 +1,82 @@
-    xhr.send(null);
-    xhr.onload=function(){
-        var data=JSON.parse(xhr.response);
+    $.getJSON(new_url,function(data){
         if(data.length==0){
-            document.getElementById("recent_archive_date").innerHTML="( Data is not available -Not archived before )";
-            document.getElementById("recent_archive_time").innerHTML="( Data is not available-Not archived before )";
-            document.getElementById("link_recent").href="https://web.archive.org/web/2/"+url;
-        }else {
+            var ids=['recent_archive_date','recent_archive_time'];
+            for(var id of ids){
+                $('#'+id).text('( Data is not available -Not archived before )');
+            }
+        }else{
             var timestamp=data[1][1];
             var date=timestamp.substring(4,6)+'/'+timestamp.substring(6,8)+'/'+timestamp.substring(0,4);
-            var time=timestamp.substring(8,10)+'.'+timestamp.substring(10,12)+'.'+timestamp.substring(12,14)
-            document.getElementById("recent_archive_date").innerHTML="( "+date+" )";
-            document.getElementById("recent_archive_time").innerHTML="( "+time+") according to Universal Time Coordinated (UTC)";
-            document.getElementById("link_recent").href="https://web.archive.org/web/2/"+url;
+            var time=timestamp.substring(8,10)+'.'+timestamp.substring(10,12)+'.'+timestamp.substring(12,14);
+            $("#recent_archive_date").text("( "+date+" )");
+            $("#recent_archive_time").text("( "+time+" ) according to Universal Time Coordinated (UTC)");
         }
-    }
+        $('#link_recent').attr('href',"https://web.archive.org/web/2/"+url);
+    });
 }
 
 function get_thumbnail(){
     var url=getUrlByParameter('url');
     url = url.replace(/^https?:\/\//,'');
     var index=url.indexOf('/');
     url=url.substring(0,index);
-    var xhr=new XMLHttpRequest();
     var new_url="https://web.archive.org/thumb/"+url;
-    xhr.open("GET",new_url,true);
-    xhr.onload=function(){
-        if(this.response.size!=233){
-            var img = document.createElement('img');
-            var url = window.URL || window.webkitURL;
-            img.src = url.createObjectURL(this.response);
-            var show_thumbnail=document.getElementById("show_thumbnail");
-            show_thumbnail.appendChild(img);
+    $.ajax({url:new_url,success:function(response){
+        if(response.size!=233){
+            $('#show_thumbnail').append($('<img src="'+new_url+'" />'));
+        }else{
+            $('#show_thumbnail').text('Thumbnail not found');
         }
-        else{
-            document.getElementById("show_thumbnail").innerHTML="Thumbnail not found";
-        }
-    }
-    xhr.onerror=function(){
-        document.getElementById("show_thumbnail").innerHTML="Thumbnail not found";
-    };
-    xhr.ontimeout = function() {
-        document.getElementById("show_thumbnail").innerHTML="Please refresh the page...Time out!!";
-    }
```

Comment:
You have dropped the old code to handle error/timeout cases but you haven't put the relevant jQuery code to handle the same cases. Its also possible in jQuery to handle these errors, please look it up and add it as well.